### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wezm**
> I'd love to get some feedback from others running and testing this just to make sure it's solid and has all features working well

I ran through the steps to get this installed on two new DigitalOcean FreeBSD droplets today. It works great and much nicer not to have a bunch of packages installed by default. Thanks for your work on this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
...
 
 Droplets built from DigitalOcean's base FreeBSD images have extra packages and scripts that will no longer be required for auto configuration. They should be disabled first to avoid conflicts. When you're confident your cleaned-up droplet is working properly with this solution, you can remove the preinstalled stuff you do not need:
+
+Unecessary packages:
+
 - avahi
 - bsd-cloudinit
 - python
 - curl
-- Related settings in /etc/rc.conf and files in /usr/local/etc
```
Comment (wezm) [inline README.md:23]: There wasn't anything to remove in `/usr/local/etc` that I found (except `sudoers` if you uninstall sudo)

File: README.md [hunk 2]
```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
...
+
+`pkg delete avahi bsd-cloudinit python27 curl`
+
+Left over users:
+- `rmuser -y avahi avahi-autoipd messagebus`
```
Comment (wezm) [inline README.md:29]: When you uninstall the packages there are messages about cleaning up these users.

File: README.md [hunk 3]
```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
...
+
+Settings in /etc/rc.conf
+- Remove ifconfig_vtnet0="dhcp"
```
Comment (wezm) [inline README.md:32]: I tried to make this more specific. This was the only thing that needed cleaning up in rc.conf... although I'm not using your auto hostname feature so maybe the hostname line should be removed too.

File: README.md [hunk 4]
```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
...
 
 Extraneous files and directories:
 - /root/.cache directory
-- selfcheck.json file
-	*(can't remember where this is found)*
```
Comment (wezm) [inline README.md:28]: I did a `locate selfcheck.json` after building the locate db and there were no matches for this file, so I removed this section.

File: README.md [hunk 5]
```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
...
 - /etc/rc.digitalocean.d directory
+- /etc/rc.d/digitalocean
```
Comment (wezm) [inline README.md:37]: This was an important one to delete to have your service take over.

**Review by wezm — COMMENTED** [review]
Some explanations of my changes.

**Comment by morganwdavis** [comment]
Nice cleanup on all this, Wesley. Thanks for the feedback.  Did you happen to try this with IPv6?

**Comment by wezm** [comment]
No I didn't test IPv6. I probably should spend the time to learn more about it.

**Pull request merged**